packages/nimble-components/src/table/template.ts

```diff
@@ -172,6 +175,12 @@ export const template = html<Table>`
                                             </slot>
                                         `)}
                                     `)}
+                                    ${repeat((x, c) => ((c.parent as Table).tableData[x.index]?.id ? (c.parent as Table).slotsByRecordId[(c.parent as Table).tableData[x.index]!.id] || [] : []), html<SlotMetadata>`
```

Comment:
this is pretty hard to understand, maybe a named method helper would be a bit clearer?

Reply:
I found a few ways to simplify this statement significantly, but I haven't figured out how to make this a named method helper without losing the observability of things.

Reply:
I talked with @atmgrifter00, and he helped me find a better way to manage this state. It's now part of the `tableData` array, so that simplified things greatly.